index.js

```diff
@@ -21,12 +21,24 @@ function parse(req) {
       label: query.label || ((query.compression ? 'gzip ' : '') + 'size'),
       color: query.color || 'brightgreen',
       style: query.style || null,
+      max: query.max || query.threshold || null,
+      threshold: query.threshold || query.max || null,
```

Comment:
With this API `max` and `threshold` are aliases. It's not clear.
`max` as a hard limit, and an optional `threshold` can be added to set a soft limit.

Basically:
```js
{
  max: query.max || null,
  threshold: query. threshold || null
}
```